apps/flame-defi/app/pool/types.ts

```diff
@@ -66,9 +73,52 @@ export type PoolContextProps = {
 
 export type PoolDetailsContextProps = {
   positionDetails: Position | undefined;
-  tokenData: TokenPriceData[];
-  percent: string;
+  poolTokenData: PoolTokenData[];
+  feeTier: string;
   symbols: string[];
   selectedSymbol: string;
   handleReverseTokenData: (symbol: string) => void;
+  collectAsWTIA: boolean;
+  handleCollectAsWTIA: (collectAsWTIA: boolean) => void;
+  txnStatus: TXN_STATUS;
+  setTxnStatus: (txnStatus: TXN_STATUS) => void;
+  modalOpen: boolean;
+  setModalOpen: (modalOpen: boolean) => void;
+  poolTokenOne: PoolTokenData;
+  poolTokenTwo: PoolTokenData;
 };
+
+export type PoolTxnStepsProps = {
+  txnStatus: TXN_STATUS;
+  poolPositionData: PoolTokenData[];
+  txnHash: string;
+  txnMsg: string;
+  addLiquidityInputValues?: string[];
+};
+
+export enum TxnType {
```

Comment:
We've got a mix of different naming conventions for enums. I started with PascalCase, but you added `TXN_STATUS`, `SWAP_INPUT_ID`, `TRADE_TYPE`, but now `TxnType` here.

We should pick one and stick with it to make them all consistent. I don't care which one.

Reply:
How about POOL_TXN_TYPE.

Reply:
Yeah, I like that! That makes sense to me.